pkg/symbol/symbol.go

```diff
@@ -201,13 +205,21 @@ func (s *Symbolizer) liner(m *pb.Mapping, path string) (liner, error) {
 // newLiner creates a new liner for the given mapping and object file path.
 func (s *Symbolizer) newLiner(buildID, path string) (liner, error) {
 	logger := log.With(s.logger, "file", path, "buildid", buildID)
-	hasDWARF, err := elfutils.HasDWARF(path)
+
+	f, err := elf.Open(path)
+	if err != nil {
+		level.Error(logger).Log("msg", "failed to open binary", "err", err)
+		return nil, ErrLinerCreationFailedFile
```

Comment:
Let's just wrap and return the error we received. No need to obfuscate the existing error. Also, this deprecates the log line above. We should only handle errors once.

Reply:
Makes sense 👍